### PR TITLE
Package blake2.0.2

### PR DESCRIPTION
**Pull request by raphael-proust**
### `blake2.0.2`
Blake2 cryptography

---
* Homepage: https://gitlab.com/nomadic-labs/ocaml-blake2
* Source repo: git+https://gitlab.com/nomadic-labs/ocaml-blake2.git
* Bug tracker: https://gitlab.com/nomadic-labs/ocaml-blake2/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/blake2/blake2.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Vincent Bernardoff <vb@luminar.eu.org>" "Nomadic Labs" ]
+license: "ISC"
+synopsis: "Blake2 cryptography"
+homepage: "https://gitlab.com/nomadic-labs/ocaml-blake2"
+bug-reports: "https://gitlab.com/nomadic-labs/ocaml-blake2/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/ocaml-blake2.git"
+build: [
+  ["dune" "build" "-j" jobs "-p" name "@install"]
```
Comment (mseri) [inline packages/blake2/blake2.0.2/opam]:
```suggestion
  ["dune" "build" "-j" jobs "-p" name]
```

That should already be the default

**Comment by mseri** [comment]
Just a minor comment, after that it is good to merge imho

**Comment by raphael-proust** [comment]
@mseri Thanks for the comment. I'm still a bit new to packaging and releasing…

**Comment by mseri** [comment]
Not a problem. Thanks for the release. New packages can be announced on the discuss.ocaml.org forum. There is an announcement tag in the community category for that purpose

**Pull request merged**